### ISSUE TITLE
Cannot install quasar from CLI "Cannot read property 'install' of undefined"

### ISSUE THREAD

**Issue by Trinovantes** [quasarframework/quasar#7848]
**Describe the bug**

I'm looking to add Quasar components into a preexisting Electron code base. However, I'm getting `Cannot read property 'install' of undefined` error when trying to install Quasar.

Here's a sample repository reproducing the bug with similar config as the project that I'm working on
https://github.com/Trinovantes/Electron-Typescript-Vue/commit/588d0123a6e08389174699ea1aa82fbdf6b0a40b

**To Reproduce**
1. `git clone https://github.com/Trinovantes/Electron-Typescript-Vue.git`
2. `yarn install`
3. `yarn dev`

**Expected behavior**
No errors in browser console

**Screenshots**
![](https://i.imgur.com/IOR8K9d.jpeg)

**Platform (please complete the following information):**
OS: Windows
Node: 12.18.3
NPM: 6.14.6
Yarn: 1.22.5
Browsers: 
iOS:
Android:
Electron: 10.1.2

**Additional context**
It's weird that Typescript has no errors so maybe the npm repository is out of sync?

**Comment by rstoenescu** [comment]
Hi,

The correct way to do it is: `import Quasar, { QTabs, ......etc } from 'quasar'`. You're using your own setup so you don't have access to the auto-import feature in Quasar CLI, which means no tree-shaking and Quasar is referenced differently. Also, you won't have to declare the components and directives (but you WILL have to declare plugins).

```js
// example with a plugin:
import Quasar, { Loading } from 'quasar'
Vue.use(Quasar, { plugins: { Loading } }

// example without any plugins
import Quasar from 'quasar'
Vue.use(Quasar)
```

**Comment by Trinovantes** [comment]
This will give a Typescript compilation error

Had to add this at the bottom of `node_modules\quasar\dist\types\index.d.ts`
```
declare const Quasar: PluginObject<Partial<QuasarPluginOptions>>
export default Quasar
```

From what I can tell, the source code is written in native JS and the Typescript definitions are declared independently (and thus leading to this desync issue where types don't match the source).

Seems the problem is located here https://github.com/quasarframework/quasar/blob/dev/ui/build/build.types.js#L398

**Comment by IlCallo** [comment]
Hey @Trinovantes, thanks for pointing this out and contributing!
However, your fix will break the named import for other usage (eg. importing Quasar object when doing jest tests setup).

The correct fix should be to export both the named value **and then** re-export it as the default

```ts
writeLine(contents, `export const Quasar: PluginObject<Partial<QuasarPluginOptions>>`)
writeLine(contents, `export default Quasar`)
```

I checked this on-the-fly and should work.
Can you check if it works for you?

**Comment by Trinovantes** [comment]
I can confirm this works for me. I've updated the PR

**Comment by rstoenescu** [comment]
Fix landing in Quasar v1.14.1 today.
Thanks for contributing!
